### PR TITLE
app-stats.json: add inProgress; add all possibilities for unit; tweak description of intervalId

### PR DESCRIPTION
**Pull request by SimonWoolf**
Bit of yakshaving on adding v3 stats to the rest api, my new test failed for schema validation issues, this fixes those and improves the descriptions a bit

### CODE REVIEW

File: json-schemas/src/app-stats.json [hunk 1]
```diff
@@ -9,17 +9,21 @@
...
+      "description": "Unit of time that this stats entry covers (partially in the case of an in-progress entry)",
+      "enum": ["minute", "hour", "day", "month"]
     },
     "schema": {
       "type": "string",
       "description": "URI to this schema."
     },
+    "inProgress": {
+      "type": "string",
+      "description": "for entries that are still in progress, such as the current month: the last sub-interval included in this entry (in format yyyy-mm-dd:hh:mm:ss), else undefined"
```
Comment (owenpearson) [inline json-schemas/src/app-stats.json:25]: For consistency:
```suggestion
      "description": "For entries that are still in progress, such as the current month: the last sub-interval included in this entry (in format yyyy-mm-dd:hh:mm:ss), else undefined"
```

File: json-schemas/src/app-stats.json [hunk 2]
```diff
@@ -9,17 +9,21 @@
...
     "intervalId": {
       "type": "string",
-      "description": "time period for statistics in date-time format yyyy-mm-dd:hh:mm:ss"
+      "description": "Start time for this stats entry. Format is yyyy-mm-dd:hh:mm for unit=minute, and yyyy-mm-dd:hh for unit=hour,day,month"
     },
     "unit": {
       "type": "string",
-      "description": "Unit of time for these statistics from the intervalId forwards.",
-      "pattern": "^minute$"
+      "description": "Unit of time that this stats entry covers (partially in the case of an in-progress entry)",
```
Comment (owenpearson) [inline json-schemas/src/app-stats.json:16]:
```suggestion
      "description": "Unit of time that this stats entry covers (partially, in the case of an in-progress entry)",
```

**Review by owenpearson — APPROVED** [review]
LGTM, couple of minor suggestions

**Review by lmars — COMMENTED** [review]
This should [bump the schema version](https://github.com/ably/ably-common/tree/main/json-schemas#versioning)

**Pull request merged**